modules/nodes-cma-autoscaling-custom-prometheus-config.adoc

```diff
@@ -171,23 +180,32 @@ $ oc create -f <file-name>.yaml
 [source,yaml]
 ----
 apiVersion: rbac.authorization.k8s.io/v1
-kind: RoleBinding
+kind: <binding_type> <1>
```

Comment:
based on the other portions of this procedure where you are using `TriggerAuthentication` method parameters, i think it makes sense to stick with `RoleBinding` to be consistent